### PR TITLE
debian/control: Improved Debian Package descriptions

### PR DESCRIPTION
**Pull request by blattms**
For the proposal of OPM as a debian package (see issue #523). Comments/further improvements welcome.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -43,31 +53,50 @@ Multi-Arch: same
...
-Description: opm grid library -- documentation
- opm-grid is intended to enable working with corner-point or, more
- generally, pillar grids in dune. A standard grid type in the petroleum
- industry, corner-point grids fill space with a relatively low number
- of cells while still providing sufficient flexibility to model faults,
- fractures and erosion. The grid format was originally designed with
- an eye towards geological modelling rather than numerical simulation
- and this design choice does limit the number of feasible numerical
- methods.
+Description: DUNE grid implementations for reservoir simulation -- library
```
Comment (akva2) [inline debian/control:81]: documentation.

File: debian/control [hunk 2]
```diff
@@ -76,16 +105,27 @@ Architecture: any
...
-Description: opm grid library -- applications
- opm-grid is intended to enable working with corner-point or, more
- generally, pillar grids in dune. A standard grid type in the petroleum
- industry, corner-point grids fill space with a relatively low number
- of cells while still providing sufficient flexibility to model faults,
- fractures and erosion. The grid format was originally designed with
- an eye towards geological modelling rather than numerical simulation
- and this design choice does limit the number of feasible numerical
- methods.
+Replaces: corner point grid utility programs.
```
Comment (akva2) [inline debian/control:108]: missing Description:

Comment (blattms) [reply debian/control]: fixed.

File: debian/control (hunk 1 above)
Comment (blattms) [reply debian/control]: fixed

**Review by blattms — COMMENTED** [review]
thanks a lot. Should be fixed now.

**Comment by blattms** [comment]
Closing as the plan should be to use the effort on https://salsa.debian.org/science-team/opm-grid in the long run

**Pull request closed without merging**